.github/release.yaml

```diff
@@ -0,0 +1,43 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Provide the configuration and categorization for generated release notes.
+
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+
+  categories:
+    - title: Proposals Accepted and Merged 📜
```

Comment:
Minor, but you might consider "Proposals accepted and merged" for our documentation style

Reply:
Sure, done.